11-payment-encoding.md

```diff
@@ -43,7 +43,7 @@ and MUST fail if the checksum is incorrect.
 The human-readable part of a Lightning invoice consists of two sections:
 1. `prefix`: `ln` + BIP-0173 currency prefix (e.g. `lnbc` for bitcoin mainnet, `lntb` for bitcoin testnet and `lnbcrt` for bitcoin regtest)
 1. `amount`: optional number in that currency, followed by an optional
-   `multiplier` letter
+   `multiplier` letter. The unit encoded here is the 'social' convention of a payment unit -- in the case of Bitcoin it is 'bitcoin' NOT satoshis.
```

Comment:
Well you can't have a fraction of a satoshi, but I think you're saying here it should be the largest unit socially accepted?

Reply:
Isn't there an implicit assumption here that there is a 'base unit', and the multiplier is applied to that base unit? I guess I'm trying to explicitly say what the base unit is.

Reply:
Also you can't have a fraction of a satoshi, but in the current scheme it is possible to encode an invoice with a fraction of a satoshi no? `1p`?

Reply:
You can pay invoices of `mSAT` value.